cd2h_repo_project/modules/theme/static/scss/recordcreationpage.scss

```diff
@@ -34,16 +35,16 @@
       padding-top: 5px;
 
       .sort-handle {
-          cursor: move;
+        cursor: move;
       }
 
-      .close {
-        padding-left: 8px;
-      }
     }
   }
+}
 
-  .authors-list .schema-form-section {
+
+.authors {
+  .schema-form-section {
```

Comment:
Do these two classes get created dynamically? I could not find them hardcoded in the html anywhere.